lib/core/kd.coffee

```diff
@@ -21,6 +21,8 @@ module.exports =
       if @[key] then throw new Error "#{key} is already registered"
       else @[key] = val
 
+  deleteSingletons: -> singletons = []
```

Comment:
you probably meant `singletons = {}` ?
https://github.com/koding/kd/pull/163/files#diff-2e2f9603b796166555ec8807de005c9bR6